scripts/upload_zoom_recordings.py

```diff
@@ -87,9 +84,7 @@ def download_file(url, download_path, query=None):
 
     return filepath
 
-DO_FILTER = False
-
-youtube = get_youtube_client()
+youtube = get_youtube_client(YOUTUBE_CREDENTIALS_FILE)
 with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
While you’re here refactoring, it can be helpful to wrap this code in a function and call it only if being loaded as a script rather than imported as a library:

```py
def main():
    # All the main program logic here

if __name__ == '__main__':
    main()
```

That way it’s easier to write tests later, because you can `import` this module in your tests without causing the main program logic to immediately execute.